MovieAppPetProject/MovieAppPetProject/GetMovieDetailsByID/GetMovieDetails-ViewController.swift

```diff
@@ -8,10 +8,35 @@
 import UIKit
 
 class MovieDetailsViewController: UIViewController {
-    private let movieDetailsViewModel = MovieDetailsViewModel()
-
+    private var selectedSearchMovie: SearchMoviesResults?
```

Comment:
Stick to a pattern. I suggest all IOutlets first then all other variables.

public vars then private vars. same as public funcs then private funcs.

include // MARKS.

e.g. // MARK IBOutlets

 @IBOutlet weak var titleLabel: UILabel!
...

// MARK: Variables

private var selectedHomeScreenMovies: NowPlayingResults?

Reply:
understood